src/otx/algorithms/anomaly/tasks/openvino.py

```diff
@@ -413,7 +409,7 @@ def deploy(self, output_model: ModelEntity) -> None:
 
         task_type = str(self.task_type).lower()
 
-        parameters["type_of_model"] = task_type
+        parameters["type_of_model"] = "AnomalyDetection"
```

Comment:
deploy only works at `AnomalyDetection`?

Reply:
Since the base anomaly models are the same for all the tasks, we just use the `task_type` parameter to modify the post-processing. See this line for example https://github.com/openvinotoolkit/model_api/blob/ca5a91ed5b3dbf428dc4de6b72f0a3da93d2aa0a/model_api/python/openvino/model_api/models/anomaly.py#L70

Reply:
Thanks for the clarification